Libplanet/Action/ActionEvaluator.cs

```diff
@@ -823,5 +766,50 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
 
             return (accountStateGetter, accountBalanceGetter, totalSupplyGetter);
         }
+
+        private IEnumerable<IAction> CreateCustomActions(
+            IPreEvaluationBlockHeader blockHeader,
+            ITransaction tx
+        )
+        {
+            if (tx.CustomActions is { } customActions)
+            {
+                IDictionary<string, Type> types = _actionTypeLoader.Load(blockHeader);
```

Comment:
Could we cached those type until load `ActionEvaluator<T>`? IMO those action types dose not change until re-start node or client.

Reply:
Assuming `StaticActionTypeLoader`, it's true. but we want to change that type upon the block header (or other contextual information) and... it's the main motivation of this work.

Reply:
Also, since the current `StaticActionTypeLoader` statically loads the type at class load time and returns it as it is, no performance loss is expected.

Reply:
> we want to change that type upon the block header (or other contextual information)

Aha, I missed that part. Yeah, you're right.